### ISSUE TITLE
0.9.3 broken: scripts folder not found

### ISSUE THREAD

**Issue by Kriechi** [beerfactory/hbmqtt#143]
I just tried to install the latest release, but it fails to start hbmqtt:
```
Traceback (most recent call last):
  File "/usr/local/bin/hbmqtt", line 7, in <module>
    from scripts.broker_script import main
ImportError: No module named 'scripts'
```

Installing 0.9.2 works as expected.
Installed on Debian Stretch, pip3 9.0 on py3.5.

**Comment by twogearvalve** [comment]
I experience the same behaviour when trying to launch hbmqtt on macOS 10.13.6 (installed using: pip3 install hbmqtt)
As a temporary workaround I copied the "scripts" folder to my site-packages manually.

**Comment by Kriechi** [comment]
@njouanin any chance to re-package the last release and include the missing files?

**Comment by njouanin** [comment]
Sorry, for this bug. This was due to the `scripts/__init__.py` which has been deletes by a previous commit.
=> Fixed in 0.9.4
